Makefile

```diff
@@ -29,8 +37,13 @@ ASM_PROC_FORCE ?= 0
 N_THREADS ?= $(shell nproc)
 # MIPS toolchain prefix
 MIPS_BINUTILS_PREFIX ?= mips-linux-gnu-
+# Python virtual environment
+VENV ?= .venv
 # Python interpreter
-PYTHON ?= python3
+PYTHON ?= $(VENV)/bin/python3
```

Comment:
Something interesting that I learnt a few days ago when debugging a bug of a python library that only happened on Windows (it was a blast as you can imagine), is that the Python binary and activate script are not put under `venv/bin/`, but under the `venv/Scripts/` path instead. https://docs.python.org/3/library/venv.html#how-venvs-work

Sounds like it could be a problem for Windows users not using WSL

Reply:
Latest commits should fix this. Note I moved the os detection before the defaults, so PYTHON would still be editable in .make_options.